bazel/envoy_test.bzl

```diff
@@ -123,7 +123,7 @@ def envoy_cc_fuzz_test(name, corpus, deps = [], tags = [], **kwargs):
         testonly = 1,
         data = [corpus_name],
         deps = [":" + test_lib_name],
-        tags = ["manual"] + tags,
+        tags = ["manual", "libfuzzer"] + tags,
```

Comment:
Nit: I think `fuzz` is a better tag name, as we will use this to generally identify fuzz targets.

Reply:
`fuzz` seems to catch all fuzz targets, driverless and test binaries (even though querying for the BUILD file for these doesn't list it as one of the tags)

Reply:
OK, then some other name is good as well, as long as it's not libfuzzer specific :)

Reply:
Should be `fuzzer` now! I think git just didn't show the updated one